ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vxlan_interface.schema.yml

```diff
@@ -19,6 +19,9 @@ keys:
               source_interface:
                 type: str
                 description: Source Interface Name
+              controller_client:
```

Comment:
Looking at the CLI options, I am wondering if we should make controller_client a dict with an enabled key + leave room for the import vlan later? otherwise we will need to have a ket like `controller_client_import_vlan` later on.

```
interface Vxlan1
   vxlan source-interface Loopback42
   vxlan controller-client import vlan 42
   vxlan controller-client
   vxlan udp-port 4789
Leaf1(config-if-Vx1)#no vxlan controller-client
Leaf1(config-if-Vx1)#show active
interface Vxlan1
   vxlan source-interface Loopback42
   vxlan controller-client import vlan 42
   vxlan udp-port 4789
```

```
Leaf1(config-if-Vx1)#show cli commands  | grep controller-cli
[no|default] vxlan controller-client
[no|default] vxlan controller-client import vlan ( ( none ) | ( remove IMPORT_DISABLED_VLANS ) | ( [ add ] IMPORT_ENABLED_VLANS ) )
```

Reply:
fixed